revolv/static/css/screen.css

```diff
@@ -3700,6 +3702,181 @@ header.edit-project-header > .container > h1 {
   line-height: 24px;
 }
 
+/*mystyle start 29-june here*/
```

Comment:
I'm sorry that you've had to deal with this HUGE CSS file (it was committed by another developer when we didn't have a review process).

A comment here is a good idea, but the comment `mystyle start 29-june here` does not describe anything. Please update it to describe what the styles below are for. For example:

```css
/**
 * Styles for ***description of the feature*** start below.
 */
```